kubernetes-deployment/maintaintool/add-worker-node.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation
+# All rights reserved.
+#
+# MIT License
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+# documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+# to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+# BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+# DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+
+# Edit /etc/hosts file for remote host
+
+# set host ip
+hostip=$1
+
+# Change 127.0.0.1 line to "127.0.0.1 localhost $hostname"
+grep -q "127.0.0.1" /etc/hosts && \
+    sed -i "/127.0.0.1/c\127.0.0.1 localhost $(hostname)" /etc/hosts || \
+    sed -i "\$a127.0.0.1 localhost $(hostname)" /etc/hosts
+
+# Comment 127.0.1.1 line
+sed -i "/127.0.1.1/s/^/# /" /etc/hosts
+
+# Comment hostip line
+sed -i "/$hostip/s/^/# /" /etc/hosts
+
+# Change hostip line to "$hostip $hostname"
+# grep -q "$hostip" /etc/hosts && \
+#     sed -i "/$hostip/c\\$hostip $(hostname)" /etc/hosts || \
+#     sed -i "\$a$hostip $(hostname)" /etc/hosts
+
```

Comment:
Will this lines possibly used in future? If not, please consider to delete them. If yes, please comment why comment it out.